WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -72,7 +72,7 @@ enum FeatureFlag: Int, CaseIterable, OverrideableFlag {
         case .siteIconCreator:
             return BuildConfiguration.current != .appStore
         case .weeklyRoundup:
-            return true
+            return AppConfiguration.isJetpack
```

Comment:
I've followed the convention for blogging prompts here

Reply:
Maybe it would be enough to disable the `FeatureFlag` to turn off this feature for Jetpack, without making any additional changes?

Reply:
Note: It should be changed if we don't remove this feature entirely from WordPress